RuleDocumentation/AvoidShouldContinueWithoutForce.md

```diff
@@ -6,7 +6,8 @@
 
 Functions that use ShouldContinue should have a boolean force parameter to allow user to bypass it.
 
-You can get more details by running `Get-Help about_Functions_CmdletBindingAttribute` and `Get-Help about_Functions_Advanced_Methods` command in Windows PowerShell.
+You can get more details by running `Get-Help about_Functions_CmdletBindingAttribute` and
+`Get-Help about_Functions_Advanced_Methods` command in Windows PowerShell.
```

Comment:
nitpicky but when i think Windows PowerShell I think PS v5.x and so not sure if worth clarifying it works in PS 7.1. Just a suggestion

Reply:
I removed "Windows" from "Windows PowerShell" in most places. I missed this one. I will fix it.

Reply:
Fixed